libsoftwarecontainer/src/container.cpp

```diff
@@ -516,6 +516,24 @@ ReturnCode Container::destroy(unsigned int timeout)
     return ReturnCode::SUCCESS;
 }
 
+ReturnCode Container::bindMountInContainer(const std::string &pathOnHost,
```

Comment:
Throughout it's path-ON-host and path-IN-container. Any reason for them to be different w.r.t. "in" and "on"?